gravitee-am-repository/gravitee-am-repository-mongodb/src/main/java/io/gravitee/am/repository/mongodb/management/AbstractUserRepository.java

```diff
@@ -67,9 +69,13 @@ public abstract class AbstractUserRepository<T extends UserMongo> extends Abstra
     protected static final String FIELD_EMAIL = "email";
     protected static final String FIELD_EMAIL_CLAIM = "additionalInformation.email";
     protected static final String FIELD_EXTERNAL_ID = "externalId";
+    private static final String OLD_INDEX_REFERENCE_TYPE_REFERENCE_ID_USERNAME_SOURCE = "referenceType_1_referenceId_1_username_1_source_1";
```

Comment:
I found it confusing to name the constant `OLD_`. If I understand correctly, the index will be deleted and re-created using the same name but with an extra constraint

Reply:
done, "OLD_" removed from the name

Reply:
Class `MongoUserProvider` also has a constant named `OLD_` 😉

Reply:
updated :D